app.js

```diff
@@ -0,0 +1,152 @@
+
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/3watermark.png',
+    id: 'third-container',
+    idBtn: 'notsee2',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Website Protfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/watermarker.png',
+    id: 'forth-container',
+    idBtn: 'notsee3',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Profesional Art Printing Data More',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/Blurwatermark.png',
+    id: 'fifth-container',
+    idBtn: 'notsee4',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Data Dashboard Healthcare',
+
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/3watermark.png',
+    id: 'sixth-container',
+    idBtn: 'notsee5',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Website Protfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+
+];
+
+function renderCards(i) {
+  const div = document.createElement('div');
+  div.className = 'middle-1-container1';
+
+  div.innerHTML = `<div id="${projects[i].id}" class="middle-1-container1">
+        <h2 class="mobile">${projects[i].name}</h2>
+      <h3 class="desktop">${projects[i].name1}</h3>
+        
+        
+        <p>${projects[i].description}</p>
+  
+        <ul class="cardbtn">
+              ${projects[i].technologies.map((tech) => `<li><button type="button" id="${projects[i].idSmallbtn}" class="middle-1-container1-btn">${tech}</button></li>`).join('')}
+        </ul>
+        <button type="button" id="${projects[i].idBtn}" onclick="openPopup()" class="see-button-one1">See project</button>
+      </div>`;
+  // eslint-disable-next-line prefer-const
+  let nameid = projects[i].id;
+
+  // eslint-disable-next-line prefer-template
+  div.querySelector('#' + nameid).style.backgroundImage = `url("${projects[i].url}")`;
+  document.querySelector('.middle-section').appendChild(div);
+}
+
+// eslint-disable-next-line no-unused-vars
+function openPopup() {
```

Comment:
- [ ] Please make sure that you are respecting the original mock-up, as you can see in the screenshot below you are missing some elements in your modal, such as `title`, `See source`

Current design            |  original design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/163489840-49ba38f1-530d-4f13-8e8f-1aa9790bac0b.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/163489835-9a3f9a72-bd2b-476d-aa36-0b23b4b7f326.PNG)

Reply:
- [ ] As suggested by the previous reviewer, kindly make sure that you are following the original design. Make sure to follow the spacing as per the design.

- Your design:
<img width="975" alt="Screenshot 2022-04-15 at 3 11 06 AM" src="https://user-images.githubusercontent.com/70269507/163496297-d918fd57-e260-497c-86fb-59562ae0c3c4.png">

- Figma design:
<img width="504" alt="Screenshot 2022-04-15 at 3 15 43 AM" src="https://user-images.githubusercontent.com/70269507/163496337-db73747f-91dd-41c8-ba81-c9d719d97469.png">

Reply:
- [ ] Kindly reduce the space between title and list as shown in the image, following the Figma design.
<img width="886" alt="Screenshot 2022-04-15 at 9 42 47 AM" src="https://user-images.githubusercontent.com/70269507/163532483-4c17fafe-d033-406b-b225-4d8f30a3885f.png">